src/views/studio/studio-comments.jsx

```diff
@@ -74,54 +75,67 @@ const StudioComments = ({
                 <h2><FormattedMessage id="studio.commentsHeader" /></h2>
                 {canEditCommentsAllowed && <StudioCommentsAllowed />}
             </div>
-            <div>
-                {shouldShowCommentComposer ?
-                    (commentsAllowed ?
-                        <ComposeComment
+            {!hasFetchedSession || studioCommentsGloballyEnabled ?
```

Comment:
Why is the `!hasFetchedSession` here blocking the comments from rendering? possibly related to my other comment about not blocking comment loading with session request